XIVComboPlugin/IconReplacer.cs

```diff
@@ -764,7 +764,28 @@ private ulong GetIconDetour(byte self, uint actionID)
                 }
 
             // MONK
-            // haha you get nothing now
+            if (Configuration.ComboPresets.HasFlag(CustomComboPreset.MonkFuryCombo))
+            {
+                var gauge = JobGauges.Get<MNKGauge>();
```

Comment:
I'm not sure what level of caching is happening for getting gauges, but I'd prefer to only grab this gauge for an action that it applies to. Not every single time this function runs if the combo is set.

Reply:
Oops! You're right. I actually had the same thought myself, but I guess it slipped my mind by the time I was done. Switched it around in my recent commit, so it should be good now.